packages/web/src/js/Modal.ts

```diff
@@ -75,10 +75,9 @@ class Modal extends BaseComponent {
     }
 
     const target = SelectorEngine.findOne(event.currentTarget.dataset.target as string);
-    const modalEl = target.parentElement.parentElement;
+    const modalEl = target?.parentElement?.parentElement;
     const toggleEl = SelectorEngine.findOne(MODAL_TOGGLE_SELECTOR, modalEl);
-    target.showModal();
```

Comment:
The modal component now doesn't work. I think you shouldn't remove this line. Can we test this?

Reply:
Fixed. This was unintentional because I do not know the `showModal` method on HTMLDialogElement :-)